wakuv2/waku_test.go

```diff
@@ -35,13 +34,14 @@ import (
 	"github.com/status-im/status-go/wakuv2/common"
 )
 
-var testENRBootstrap = "enrtree://AL65EKLJAUXKKPG43HVTML5EFFWEZ7L4LOKTLZCLJASG4DSESQZEC@prod.status.nodes.status.im"
+var testENRBootstrap = "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@store.test.shards.nodes.status.im"
```

Comment:
Maybe better to use ENR for shards.staging here and also so that bootstrap nodes also can be used in other tests. 
Maybe for store test specifically you can override with this one

Reply:
I had fixed some of these tests as part of https://github.com/status-im/status-go/pull/4665/ . I was wondering if there were others using defaultPubsubtopic.